packages/react-reconciler/src/ReactFiber.new.js

```diff
@@ -56,7 +56,7 @@ import {
   Block,
   OffscreenComponent,
   LegacyHiddenComponent,
-} from './ReactWorkTags';
+} from 'react-reconciler/src/ReactWorkTags';
```

Comment:
Why change relative to absolute when we're in the same directory?

Reply:
sorry, I did an overly-zealous find&replace in CLion/IntelliJ. I'll fix these.